cmd/kubeadm/app/phases/addons/proxy/manifests.go

```diff
@@ -112,5 +112,7 @@ spec:
       - key: CriticalAddonsOnly
         operator: Exists
       - operator: Exists
+      nodeSelector:
```

Comment:
I'm not familiar with windows deployments, but how does service routing work on window then?

Reply:
it works somehow, this is more of a question for the SIG Window folk.

Reply:
@michmike ^

Reply:
kube-proxy still runs and handles service routing, it's just that it needs to run on the host directly and can't be run from inside a privileged container.